SingularityUI/app/collections/TaskSearchResults.coffee

```diff
@@ -0,0 +1,63 @@
+Collection = require './collection'
```

Comment:
Looking at our other collections/models, it seems we are doing some double work with this and the `RequestHistoricalTasks` collection. Both are the same url, this one has the extra params, the other has a few more methods. I would argue that we merge the two together (maybe into `RequestHistoricalTasks` for naming). `RequestHistoricalTasks` also already has a `TaskHistoryItem` defined that we can use in place of the `TaskSearchResult` model